presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -98,7 +98,7 @@ public static long currentTime(ConnectorSession session)
     {
         // Stack value is number of milliseconds from start of the current day,
         // but the start of the day is relative to the current time zone.
-        long millis = getChronology(session.getTimeZoneKey()).millisOfDay().get(session.getStartTime());
+        long millis = UTC_CHRONOLOGY.millisOfDay().get(session.getStartTime());
```

Comment:
technically, you should fix or erase comment above. I can't make sense of it before than change, but the more so afterwards. Note: this commit is clobbered in "Fix current_time, localtime & localtimestamp semantics" commit, erasing here will produce trivial rebase conflict later.